Source/WebCore/html/canvas/WebGL2RenderingContext.cpp

```diff
@@ -3111,7 +3133,7 @@ WebGLAny WebGL2RenderingContext::getParameter(GCGLenum pname)
     case GraphicsContextGL::TRANSFORM_FEEDBACK_BUFFER_BINDING:
         return m_boundTransformFeedbackBuffer;
     case GraphicsContextGL::TRANSFORM_FEEDBACK_BINDING:
-        return m_boundTransformFeedback == m_defaultTransformFeedback ? nullptr : m_boundTransformFeedback;
+        return m_boundTransformFeedback == m_defaultTransformFeedback ? nullptr : RefPtr { m_boundTransformFeedback.get() };
```

Comment:
Does the `operator RefPtr<T>` on `WebGLBindingPoint<T>` not kick in here?

Reply:
I don't know how it would be possible since a) the return value is WebGLAny and b) tertiary operator returning nullptr
e.g. the typing is
`WebGLAny = nullptr_t or WebGLBindingPoint<WebGLTransformFeedback, ...> `
I think there's too many conversions to reach the RefPtr< WebGLTransformFeedback> inside WebGLAny, but I never remember all the intricacies of the implicit conversions. Certainly it had a compile error..